pyproject.toml

```diff
@@ -12,7 +12,7 @@ authors = [
 description = "Training Library"
 readme = "README.md"
 license = {text = "Apache-2.0"}
-requires-python = ">=3.11"
+requires-python = ">=3.9"
```

Comment:
not called out in the commit message (oops) -- but this is to match the dependencies of the CLI

Please let me know if there's a reason this lib can't support < 3.11